components/Feedback/BubbleChat/index.tsx

```
@@ -39,17 +38,21 @@ export const BubbleChat = ({
       return !inline && match ? (
-        <div className="rounded-md bg-gray-900 p-4">
+        <div className="rounded-md p-4">
           <SyntaxHighlighter
             language={match[1]}
             style={vscDarkPlus}
+            customStyle={{
+              borderRadius: "0.5rem",
+              padding: "1rem",
+            }}
             PreTag="div"
             children={String(children).replace(/\n$/, "")} // eslint-disable-line react/no-children-prop
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)